commit 6d21b760b7a6603a16216ba78d2b6457e12ad6e6
Author: Geert Uytterhoeven <geert+renesas@glider.be>
Date:   Tue Jul 30 16:32:24 2024 +0200

    ARM: shmobile: defconfig: Enable slab hardening and kmalloc buckets
    
    Enable slab freelist metadata hardening, which in turn enables support
    for allocation from separate kmalloc buckets.  The latter is recommended
    to be enabled, as per its help text.
    
    Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Link: https://lore.kernel.org/96fc3f0ed509b6364d8aeb297d5acf4abed18404.1722349907.git.geert+renesas@glider.be

diff --git a/arch/arm/configs/shmobile_defconfig b/arch/arm/configs/shmobile_defconfig
index 56925adfe842..0e380e450a62 100644
--- a/arch/arm/configs/shmobile_defconfig
+++ b/arch/arm/configs/shmobile_defconfig
@@ -24,6 +24,7 @@ CONFIG_CPUFREQ_DT=y
 CONFIG_VFP=y
 CONFIG_NEON=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
+CONFIG_SLAB_FREELIST_HARDENED=y
 CONFIG_CMA=y
 CONFIG_NET=y
 CONFIG_PACKET=y